push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/PushNotificationSender.java

```diff
@@ -27,6 +27,8 @@
  */
 public interface PushNotificationSender {
 
+    String AEROGEAR_PUSH_ID = "aerogear-push-id";
```

Comment:
not sure, this is a header name, while the interface is the boundsry for
dispatching

we can keep it here, but nit sure the location is that good

On Wednesday, April 29, 2015, Sebastien Blanc <notifications@github.com>
wrote:

> In
> push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/PushNotificationSender.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/540#discussion_r29349251>
> :
>
> > @@ -27,6 +27,8 @@
> >   */
> >  public interface PushNotificationSender {
> >
> > +    String AEROGEAR_PUSH_ID = "aerogear-push-id";
>
> Where should that be ? I thought it was a good place
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/540/files#r29349251>
> .
>


-- 
Sent from Gmail Mobile